app/uk/gov/hmrc/agentservicesaccount/views/pages/asa_dashboard.scala.html

```diff
@@ -45,7 +45,12 @@
     @p(msgs("agent.accountNumber", arn))
 
     @client_authorisations()
-    @manage_section()
+
+    @if(isAdmin) {
+        @manage_section()
+    }else{
+        @None
```

Comment:
you can just do the if case and not the else in a template.

Reply:
Yes